runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -127,22 +129,24 @@ static StreamExecutionEnvironment createStreamExecutionEnvironment(
     LOG.info("Creating a Streaming Environment.");
 
     String masterUrl = options.getFlinkMaster();
-    Configuration flinkConfig = getFlinkConfiguration(confDir);
+    Configuration flinkConfiguration = getFlinkConfiguration(confDir);
```

Comment:
this change is for consistency with the batch environment creation function (also in this file)

Reply:
I don't think this change is necessary for the scope of this PR.